src_server/check_carrier.sh

```diff
@@ -4,14 +4,10 @@ BOOTSTRAP_FILE=${CARRIER_PATH}/bootstrapd.conf
 CURRENT_IP=$(curl -s ifconfig.me)
 echo "$CURRENT_IP"
 CARRIER_IP=$(grep "external_ip" ${BOOTSTRAP_FILE} | cut -d'"' -f 2)
-echo $CARRIER_IP
-
-if [ "$CURRENT_IP" == "" ]; then
+echo $CARRIER_IPif [ "$CURRENT_IP" == "" ]; then
```

Comment:
@Firiyuu  theres an issue with this file. kindly test and check the console to see and issues

Reply:
@jhoe123 I think I messed this up as I tried debugging. I revised it back to original, I didnt meant to change any of check_carrier.sh